app/src/main/java/kotlinFifthTask/firstTask.kt

```diff
@@ -0,0 +1,17 @@
+package kotlinFifthTask
+
+/**
+ *      Задание 1
+ *
+ *      Создайте список чисел от 1 до 10, затем используйте метод filter
+ *      чтобы оставить только четные числа. После этого используйте метод map
+ *      чтобы умножить каждое число на 2.
+ */
+
+val numbers = listOf(1, 2, 3, 4, 5, 6, 7, 8, 9, 10)
+
+val newList = numbers.filter { it % 2 == 0 }.map { it * 2 }
```

Comment:
```suggestion
val newList = numbers
    .filter { it % 2 == 0 }
    .map { it * 2 }
```
в котлине так принято делать такие операции

Reply:
спасибо, поменяю